pkg/catalog/search.go

```diff
@@ -5,34 +5,22 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
-)
-
-// HTTPClient defines an http client which then can be used to test the
-// handler code.
-//go:generate counterfeiter -o fakes/fake_http_client.go . HTTPClient
-type HTTPClient interface {
-	Do(*http.Request) (*http.Response, error)
-}
 
-var httpClient HTTPClient = http.DefaultClient
+	"github.com/weaveworks/profiles/api/v1alpha1"
```

Comment:
as part of https://github.com/weaveworks/profiles/pull/47 we started to use a similar pattern to `helmv2` and `sourcev1`, so `profilev1` instead of `v1alpha1`, could we change that here too?